Missions/ProjectApollo/Apollo 10.cfg

```diff
@@ -13,4 +13,34 @@ LMPVesselName=Gene Cernan
 CDRSuitName=T. STAFFORD
 CMPSuitName=J. YOUNG
 LMPSuitName=E. CERNAN
-CSMCueCard=1 SATURN_BOOST_A14
\ No newline at end of file
+CSMCueCard=1 SATURN_BOOST_A14
+CMPIPABIASX=-0.27
+CMPIPASCALEX=-100
+CMPIPABIASY=-0.07
+CMPIPASCALEY=-230
+CMPIPABIASZ=0.05
+CMPIPASCALEZ=-80
+CMNBDX=0.4
+CMNBDY=-1.3
+CMNBDZ=1.2
+CMADIAX=1
+CMADIAY=13
+CMADIAZ=11
+CMADSRAX=10
+CMADSRAY=-1.3
+CMADSRAZ=7
+LMPIPABIASX=-0.41
+LMPIPASCALEX=-430
+LMPIPABIASY=0.18
+LMPIPASCALEY=-840
+LMPIPABIASZ=-0.03
+LMPIPASCALEZ=-530
+LMNBDX=-3.2
+LMNBDY=1.5
+LMNBDZ=1.2

```

Comment:
Should be -1.2 and not 1.2, see flown padload.

Reply:
You changed CMNBDZ from 1.2 to -1.2, when you should have changed LMNDBZ from 1.2 to -1.2. CMNBDZ should be positive, LMNDBZ negative. Sadly they have the same value, just different signs, for maximum confusion.